[Bug] Fresh OR clone doesn't compile

### Describe the bug

After cloning OR, it fails to compile because the database child project isn't retrieved. Retrieving it manually gets the build to work.

### To Reproduce

1. git clone git@github.com:openrocket/openrocket.git
2. ./gradlew shadowJar
FAILURE: Build failed with an exception.

* Where:
Build file '/home/joseph/rockets/openrocket/git/openrocket/unstable/core/build.gradle' line: 160

* What went wrong:
Execution failed for task ':core:submoduleInit'.
> Could not find method exec() for arguments [build_19kuh11xq3s5o3vu0wiob300t$_run_closure15$_closure30$_closure31@c56f1d4] on task ':core:submoduleInit' of type org.gradle.api.DefaultTask.
3. git submodule init
4. git submodule update --remote
5. ./gradlew shadowJar
BUILD SUCCESSFUL in 40s

...


### Screenshots / .ork file

#### Screenshot(s):
*(drag-and-drop the screenshot(s) here)*


#### .ork file:
*(drag-and-drop the file here as a .zip file)*


### OpenRocket version

unstable

### What platform are you running on?

Linux

### Additional context

_No response_

--- a/core/build.gradle
+++ b/core/build.gradle
@@ -155,27 +155,18 @@ def externalComponentsPath = './src/main/resources/datafiles/components/database
 def externalResourceComponents = './resources-src/datafiles/openrocket-database'
 
 // Initializes the submodules in Git.
-tasks.register('submoduleInit') {
-    doLast {
-        exec {
-            workingDir "../"
-            executable 'git'
-            args 'submodule', 'init'
-        }
-        println "Executed submoduleInit!"
-    }
+
+// Initializes the submodules in Git.
+tasks.register("submoduleInit", Exec) {
+    workingDir = "../"
+    commandLine("git", "submodule", "init")
 }
+
 // Updates the initialized submodules in Git.
-tasks.register('subModuleUpdate') {
+tasks.register("subModuleUpdate", Exec) {
     dependsOn 'submoduleInit'
-    doLast {
-        exec {
-            workingDir "../"
-            executable 'git'
-            args 'submodule', 'update', '--remote'
-        }
-        println "Executed subModuleUpdate!"
-    }
+    workingDir = "../"
+    commandLine("git", "submodule", "update", "--remote")
 }
 
 // Copies the external components to the Core resource folder.

